modules/RTC/JitsiTrack.js

```diff
@@ -428,6 +428,10 @@ export default class JitsiTrack extends EventEmitter {
                 JitsiTrackEvents.TRACK_AUDIO_LEVEL_CHANGED,
                 audioLevel,
                 tpc);
+        } else if (this.audioLevel === 0 && audioLevel === 0 && this.isLocal()) {
```

Comment:
hm so the audio level equal to 0 means there's no input? what's the value when muted?

Reply:
LocalStatsCollector reports a value of 0.008 when the mic is muted, so we should be good here. I can add a check just in case.

Reply:
can you mention that in a comment please?

Reply:
Sure